usaspending_api/database_scripts/broker_matviews/award_category_table.sql

```diff
@@ -0,0 +1,30 @@
+create table if not exists award_category
+    (
+        type_code text not null,
+        type_name text not null
+    );
+
+create index if not exists award_category_type_code_idx on award_category using btree (type_code);
```

Comment:
Typically it best practice to create the index after the table is populated with data. Not an issue with this small table

Reply:
Will move it down.